pages/deployment/aws.mdx

```diff
@@ -0,0 +1,167 @@
+highly depends on your actuall workload. For heavy graph processing, consider
+using instances with more CPU and memory. 
+
+Memgraph, by default, stores all data in the RAM, if you are going to use the
+default storage mode (`IN_MEMORY_TRANSACTIONAL` or `IN_MEMORY_ANALYTICAL`), make
+a calculation [how much data you will
+need]((/fundamentals/storage-memory-usage#calculate-storage-memory-usage)).
+There is also an [easy-to-use
+calulator](https://memgraph.com/enterprise#ram-calculator) for approximating
+memory usage. The rule of thumb is to have double the memory of what your
+storage would take. You can go with less if you do not have a demanding
+workload.
+
+
+AWS has specialized instances for [memory-optimized
+workloads](https://docs.aws.amazon.com/ec2/latest/instancetypes/mo.html) wich
+are designed for in-memory databases, data analytics, and other memory-intensive
+applications. They also provide DDR5 memory, so you may consider using them to
+optimize for better performance. 
+
+
+### Setup the network
+
+When creating the EC2 instance, you need to set up the network access and
+security group. By default, the Memgraph uses port 7687 for the Bolt protocol.
+You need to open this port for TCP traffic and to allow connections to Memgraph. 
+
+If you change the default bolt port, make sure to update the security group
+accordingly.
+
+Also, if deploying the Memgraph for replication or clustering, the ports for the
+replication and clustering should be open as well.
+
+
+### Setup the storage 
+
+When creating the EC2 instance, you need to set up the storage. By default
+Memgraph stores all data to working RAM, but for the [persistency between
+restarts](../fundamentals/data-durability) Memgraph uses the disk storage to
+store snapshots, configurations, etc.
+
+The sizing of the storage depends on the amount of data you are going to store
+in your Memgraph instance, and the quantity of the snapshots you want to keep
+alive. 
+
+The general rule of thumb is to double the storage and then the amount of data
+you will store. If you are going to store 100GB of data, you should have at
+least 200GB of disk storage. 
+
+
+## Installing Memgraph 
+
+Depending on the way you want to deploy Memgraph, native or via Docker, you need
+to follow the steps below: 
+
+<Tabs items={["Docker" , "Linux"]}>
+
+ <Tab> 
+    For example, if you are using Amazon Linux 2, you can follow the steps
+    below to install Docker:
+
+        ```bash
+            sudo yum update -y
+            sudo yum install docker -y
+            sudo systemctl start docker
+            # Add the user to the docker group
+            sudo usermod -a -G docker $(whoami)
+            newgrp docker
+        ```
+    Note that the steps above may vary depending on your Linux distribution.
+
+    After the docker is installed, you can pull the Memgraph image and run it:
+
+        ```bash
+            docker run -p 7687:7687 memgraph/memgraph:latest
+        ```
```

Comment:
This is unnecessarily indented. (check deployment)

Reply:
Done.